api/models/v0.3/modelCard.js

```diff
@@ -134,6 +161,12 @@ class Card {
     this.STANDARD_CARD_TYPE = "standard";
   }
 
+  /**
+   * validate()
+   * Runs simple validations to make sure that the arguments are present and
+   * passes the needed requirements, and once those pass, then validates the
+   * card's address and username against the ILS.
+   */
   validate() {
     const validateByPolicy = ["email", "birthdate"];
     // These four values are necessary
```

Comment:
FOUR values?  Email, birthdate, and...?

Reply:
The comment is for the next line but I can write them down again on this line.